code/modules/jobs/job_types/serfs/innkeep.dm

```diff
@@ -61,5 +61,5 @@
 		beltr = /obj/item/reagent_containers/glass/bottle/beer/blackgoat
 		H.change_stat(STATKEY_END, 1)
 		H.change_stat(STATKEY_CON, 1)
-	backpack_contents = list(/obj/item/recipe_book/cooking)
+	backpack_contents = list(/obj/item/recipe_book/cooking, /obj/item/bottle_kit)
```

Comment:
this is probably more fitting for the soilsons or the tapster . since they are the ones with like a giant distilery